server/src/main/java/org/elasticsearch/index/cache/bitset/BitsetFilterCache.java

```diff
@@ -91,6 +91,19 @@ public BitsetFilterCache(IndexSettings indexSettings, Listener listener) {
         this.listener = listener;
     }
 
+    public static BitSet setupBitset(LeafReaderContext context, Query query) throws IOException {
```

Comment:
nit: I think swapping the parameter order makes it consistent with other method in this class.

Also, I wonder if bitSetFromQuery is a more descriptive name?

Reply:
Sounds good, changed signature and renamed :)